index.html

```diff
@@ -0,0 +1,393 @@
+
+                            <div class="col-2">
+                                <div>
+                                    <center>
+                                        <img src="images/program_icon_01.png" alt="">
+                                    </center>
+                                </div>
+                                <center>
+                                    <h3>Virtual events</h3>
+                                </center>
+                                <p>Many organizations and companies in Africa's technology sector have been turning to virtual events and conferences.</p>
+                            </div>
+                            <div class="col-2">
+                                <center>
+                                    <img src="images/program_icon_04.png" alt="">
+                                </center>
+                                <center>
+                                    <h3>Funding</h3>
+                                </center>
+                                <p>Economic downturn caused by the pandemic, many venture capitalists and angel investors have continued to invest.</p>
+                            </div>
+                            <div class="col-2">
+                                <center>
+                                    <img src="images/program_icon_03.png" alt="">
+                                </center>
+                                <center>
+                                    <h3>Collaboration</h3>
+                                </center>
+                                <p> With many companies and organizations forced to adopt remote working during the pandemic.</p>
+                            </div>
+                            <div class="col-2">
+                                <center>
+                                    <img src="images/program_icon_04.png" alt="">
+                                </center>
+                                <center>
+                                    <h3>Innovation</h3>
+                                </center>
+                                <p>Even though the pandemic has had negative effects on the economy, it has also created new opportunities for innovating</p>
+                            </div>
+                            <div class="col-2">
+                                <center>
+                                    <img src="images/program_icon_05.png" alt="">
+                                </center>
+                                <center>
+                                    <h3> support</h3>
+                                </center>
+                                <p>Governments in Africa have been investing in technology-based initiatives to support the development of the sector.</p>
+                            </div>
+
+                        </div>
+
+                    </div>
+
+
+                    <!--- mobile cards -->
+                    <div class="container cards-mobile">
```

Comment:
- [ Required ] Please consider changing this tag to the HTML5 semantic `section` tag, the `section` tag is usually used to wrap logical sections of a page that have `hx` tags, since this is a logical section of your page please wrap it in a `section` tag as shown below. 
```HTML
<section class="container cards-mobile">
  <h3> Virtual events </h3>
</section>
```